packages/astro-rss/README.md

```diff
@@ -44,6 +46,8 @@ rss({
   title: 'Buzz’s Blog',
   // `<description>` field in output xml
   description: 'A humble Astronaut’s guide to the stars',
+  // provide a base URL for RSS <item> links
+  canonicalUrl: import.meta.env.SITE,
```

Comment:
`canonicalUrl` has special meaning already in Astro, referring to the `canonical` URL of any single Astro page. It's use here as "the site root" is confusing to me as a user.

Can we call this `site` instead?

Reply:
Hm, we use `canonicalUrl` as the option for sitemap. Was using `canonicalUrl` here for consistency. Would you consider this different enough that `site` is better?

Reply:
I think so, maybe even worth changing the `canonicalUrl` sitemap option to `site` as well.

Again, only saying this because `Astro.canonicalUrl` is already such a specific thing and I wouldn't want to overload that term for Astro users.

Reply:
Understood! I'm in favor of changing 👍 Will update the docs as well